behave/features/steps/cla_eligibale_confirmation.py

```diff
@@ -0,0 +1,92 @@
+from behave import *
+from features.constants import CLA_PUBLIC_URL, CLA_NUMBER
+from features.steps.cla_in_scope import assert_header_on_page, wait_until_page_is_loaded
+
+
+def assert_form_input_element(callback_form, element_id, value):
```

Comment:
Should this function and the previous two (assert_header_on_page and wait_until_page_is_loaded) be moved to helper functions?